compiler/test/fail_compilation/fix350a.d

```diff
@@ -1,8 +1,8 @@
 /*
 TEST_OUTPUT:
 ---
-fail_compilation/fix350a.d(12): Error: comma expected separating field initializers
-fail_compilation/fix350a.d(12): Error: comma expected separating field initializers
+fail_compilation/fix350a.d(12): Error: incorrect syntax for associative array, expected `[]`, found `{}`
+fail_compilation/fix350a.d(12): Error: incorrect syntax for associative array, expected `[]`, found `{}`
```

Comment:
This is the wrong message here, this test case has nothing to do with associative arrays.